src/pycram/external_interfaces/giskard.py

```diff
@@ -314,9 +315,25 @@ def achieve_joint_goal(goal_poses: Dict[str, float]) -> 'MoveResult':
     return giskard_wrapper.execute()
 
 
+def set_joint_goal(goal_poses: Dict[str, float]) -> None:
```

Comment:
I wanted this to just set the goal without executing, I changed the method now to be a copy of achieve joint goal but without execute, and made achieve_joint_goal to call it before execute to avoid dublication.